meson.build

```diff
@@ -1540,6 +1540,10 @@ swift = get_option('swift-build').require(
 )
 features += {'swift': swift.allowed()}
 
+if features['cocoa'] and not features['swift'] and get_option('cplayer')
+    error('cplayer enabled but no suitable swift version could be found or was disabled!')
```

Comment:
It is perfectly fine, but we could rice it with meson syntax for even better diagnostic error.

``` meson
if features['cocoa']
    cplayer = cplayer.require(features['swift'], error_message: 'cplayer enabled but no suitable swift version could be found or was disabled!')
endif
```

See https://mesonbuild.com/Reference-manual_returned_feature.html#featurerequire

But either way is good, whatever you prefer.

Reply:
i think this doesn't work, because cplayer is defined as type 'boolean' in [meson_options.txt](https://github.com/mpv-player/mpv/blob/master/meson_options.txt#L3) and not 'feature'.

Reply:
I think you are right.